plugins/package-managers/cocoapods/src/main/kotlin/CocoaPods.kt

```diff
@@ -197,7 +197,6 @@ class CocoaPods(
                 "spec", "which", podspecName,
                 "--version=${id.version}",
                 "--allow-root",
-                "--regex",
```

Comment:
If https://github.com/oss-review-toolkit/ort/pull/7544#discussion_r1333130483 is true, then we should keep the `--regex` and modify the regex such that it matches exactly as proposed in the linked PR, right ?

Reply:
Yes, see https://github.com/oss-review-toolkit/ort/pull/7544#discussion_r1333153092.